### ISSUE TITLE
[New build - EXPERIMENTAL] 24w29a: New Warp core logic

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5880]
Following feedback from the community (thanks to all who contribute regularly!), the warp core logic has been changed.
Now each added pearl will fill the core existing pearl or create a new pearl of the grid scale used.
Which means that adding a small pearl will only partially fill an existing pearl or create a new partially filled one.
Likewise adding a large pearl might adds several pearl to the core.
In short there is now disadvantage of using larger grid size warp core when it come to actual fuel capacity.
